src/utils/config/index.ts

```diff
@@ -7,3 +9,34 @@
  * configuration validation.
  */
 export const APPROVEMAN_CONFIG_FILENAME = ".github/approveman.yml";
+
+export const DEFAULT_APP_ACTOR_NAME = "approveman[bot]";
+
+/**
+ * Resolve the app actor name that should be used.
+ *
+ * The app installed on GitHub, no matter with personal or
+ * orgranization account will have the app actor name as
+ * {@link DEFAULT_APP_ACTOR_NAME}, but GitHub enterprise
+ * will have a different name since the app is running in
+ * a enterprise defined environment.
+ *
+ * TODO(tianhaoz95): try to see if the actor name can be
+ * detected automatically to some degree.
+ *
+ * @returns The resolved app actor name.
+ */
+export const getAppActorName = (): string => {
+  if (process.env.GHE_HOST) {
+    if (process.env.APP_ACTOR_NAME_OVERRIDE) {
+      // eslint-disable-next-line no-magic-numbers
+      if (process.env.APP_ACTOR_NAME_OVERRIDE.length === 0) {
```

Comment:
Codacy found an issue: ['magic numbers' are not allowed: 0](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6168221)